index.html

```diff
@@ -45,17 +45,15 @@ <h1>What would you like to accomplish during this time?</h1>
               <div class="minutes">
                 <p>Minutes
                 <br />
-                <input class="minutesTextLine" type="number" min="0" max="999" onkeydown="javascript: return event.keyCode === 8 ||
-event.keyCode === 46 ? true : !isNaN(Number(event.key))" name="userInputMinutes" />
+                <input class="minutesTextLine" type="number" min="0" max="99"/>
```

Comment:
This clean up is really nice - If you want to include the `keyCode` stuff I would recommend relocating it to the javascript file.